### ISSUE TITLE
Create a script to build, test and publish the NodeJS client

### ISSUE THREAD

**Issue by evoxmusic** [Qovery/RedisLess#9]
To make RedisLess easily testable by any NodeJS developer, it would be useful to have a script to build, test and deploy the NodeJS client. Any suggestion is appreciated.

**Comment by tbmreza** [comment]
Stumbled upon this: https://dev.to/kennethlarsen/how-to-distribute-a-rust-binary-on-npm-75n

- use [this CI template](https://github.com/japaric/trust) to publish binary releases for Linux, macOS and Windows
- install.js looks something like:
```js
let exec = require('child_process').exec;

exec('curl -LSfs https://japaric.github.io/trust/install.sh | \
sh -s -- --git qovery/redisless', (error, stdout, stderr) => {
  console.log(stderr);
});
```
- node install.js

Is this close enough to what you had in mind?

**Comment by evoxmusic** [comment]
I don't think we need something like that. We already have everything we need. We just need to spend some time to automatically build and publish on NPM with Github actions. I will probably spend some time on this task once I have finished what I am working on - RedisLess cluster support
